manager/manager/utils.py

```diff
@@ -547,6 +598,12 @@ def jira_comment(request_data):
     }
     url = f"https://{os.environ.get('JIRA_API_HOSTNAME')}/rest/api/latest/issue/{jira_id}/comment"
     response = requests.post(url, json=jira_payload, headers=headers)
+
+    if "time_lost" in request_data:
+        update_time_loss(
+            jira_id=jira_id, add_time_loss=request_data.get("time_lost", 0.0)
+        )
```

Comment:
if the time loss update fails we shouldn't proceed with the rest of the process, I think we should return the response from `update_time_loss` when it returns status_code = 400.
```suggestion
    if "time_lost" in request_data:
        response = update_time_loss(
            jira_id=jira_id, add_time_loss=request_data.get("time_lost", 0.0)
        )
        if response.status_code == 400:
            return response
```